test/unit/email_rate_limit_test.rb

```diff
@@ -51,8 +58,10 @@ def seed_array
     day_ago               = rand(last_clicked_days_ago)
     clicked_ago           = valid_values.sample
     bad_clicked_ago        = invalid_values.sample
-    assert EmailRateLimit.new(day_ago).now?(clicked_ago), "Expected  EmailRateLimit.new(#{day_ago}).now?(#{clicked_ago}) to be true, was not"
-    assert EmailRateLimit.new(day_ago).skip?(bad_clicked_ago), "Expected  EmailRateLimit.new(#{day_ago}).skip?(#{bad_clicked_ago}) to be true, was not"
+    assert EmailRateLimit.new(day_ago).now?(clicked_ago),
```

Comment:
Let's fix these assertions. Let's create a new assertion, called `assert_true`, which calls `assert_equal true, argument`.

I understand wanting to avoid assert b/c the error messages are basic, but we can DRY this up.